src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final String randomColor = new ColorSupplier().getRandomColor();
+    private final int randomNumber = new Random().nextInt(100);
```

Comment:
You should have fields `randomColor` and `randomNumber` inside `getRandomBall()` method.

Reply:
Make `100` constant